site.yml.sample

```diff
@@ -37,19 +37,19 @@
       raw: sudo apt-get -y install python-simplejson
       ignore_errors: yes
       when:
-        - systempython2.stat.exists is undefined or systempython2.stat.exists == false
+        - systempython2.stat is undefined or systempython2.stat.exists is undefined or systempython2.stat.exists == false
```

Comment:
if `systempython2.stat` is defined, it means `systempython2.stat.exists` will be defined as well.
the `systempython2.stat.exists is undefined` is not necessary anymore then.